podcasts/End of Year/Stories/DummyStory.swift

```diff
@@ -0,0 +1,63 @@
+import SwiftUI
+import PocketCastsServer
+import PocketCastsDataModel
+import Kingfisher
+
+struct DummyStory: View {
+    let podcasts: [Podcast]
+
+    var backgroundColor: Color {
+        Color(podcasts.first?.bgColor() ?? UIColor.black)
+    }
+
+    var tintColor: Color {
+        .white
+    }
+
+    var body: some View {
+        ZStack {
+            backgroundColor.ignoresSafeArea()
+
+            VStack {
```

Comment:
A lot of constants here can be extracted to their own constants.

However, given that this is an early and dummy version I wonder if that's really needed.